drivers.go

```diff
@@ -4,6 +4,9 @@ import (
 	"strings"
 
 	"github.com/docker/libnetwork/driverapi"
+	"github.com/docker/libnetwork/ipamapi"
+	builtinIpam "github.com/docker/libnetwork/ipams/builtin"
```

Comment:
Good idea, it would be nice to create a directory to contain all the kind of drivers.
Let's take care of this as part of another PR. I think it is better not to mix code changes with package move changes.